packages/itwinui-variables/src/themes/light.scss

```diff
@@ -121,48 +123,49 @@ $opacities: (
   --iui-color-icon: #{hex($fg-300)};
   --iui-color-icon-hover: #{hex($fg-200)};
   --iui-color-icon-muted: #{hex($fg-700)};
   --iui-color-icon-muted-hover: #{hex($fg-600)};
   --iui-color-icon-disabled: #{hex($fg-800)};
-  --iui-color-icon-accent: var(--iui-color-icon-informational);
-  --iui-color-icon-informational: #{hex($fg-informational-2)};
-  --iui-color-icon-positive: #{hex($fg-positive-2)};
-  --iui-color-icon-warning: #{hex($fg-warning-2)};
-  --iui-color-icon-negative: #{hex($fg-negative-2)};
+  --iui-color-icon-accent: #{$fg-accent-2};
+  --iui-color-icon-informational: #{$fg-informational-2};
+  --iui-color-icon-positive: #{$fg-positive-2};
+  --iui-color-icon-warning: #{$fg-warning-2};
+  --iui-color-icon-negative: #{$fg-negative-2};
   // #endregion
   // #region Text variables
   --iui-color-text: #{hex($fg-200)};
   --iui-color-text-hover: #{hex($fg-100)};
   --iui-color-text-muted: #{hex($fg-500)};
   --iui-color-text-placeholder: #{hex($fg-650)};
   --iui-color-text-disabled: #{hex($fg-700)};
-  --iui-color-text-accent: var(--iui-color-text-informational);
-  --iui-color-text-accent-hover: var(--iui-color-text-informational-hover);
-  --iui-color-text-informational: #{hex($fg-informational-1)};
-  --iui-color-text-informational-hover: #{hex($fg-informational-0)};
-  --iui-color-text-positive: #{hex($fg-positive-1)};
-  --iui-color-text-positive-hover: #{hex($fg-positive-0)};
-  --iui-color-text-warning: #{hex($fg-warning-1)};
-  --iui-color-text-warning-hover: #{hex($fg-warning-0)};
-  --iui-color-text-negative: #{hex($fg-negative-1)};
-  --iui-color-text-negative-hover: #{hex($fg-negative-0)};
+  --iui-color-text-accent: #{$fg-accent-1};
+  --iui-color-text-accent-hover: #{$fg-accent-0};
+  --iui-color-text-informational: #{$fg-informational-1};
+  --iui-color-text-informational-hover: #{$fg-informational-0};
+  --iui-color-text-positive: #{$fg-positive-1};
+  --iui-color-text-positive-hover: #{$fg-positive-0};
+  --iui-color-text-warning: #{$fg-warning-1};
+  --iui-color-text-warning-hover: #{$fg-warning-0};
+  --iui-color-text-negative: #{$fg-negative-1};
+  --iui-color-text-negative-hover: #{$fg-negative-0};
   // #endregion
   // #region HSL variables for transparency
   --iui-color-background-hsl: #{get-hsl($bg-0)};
   --iui-color-foreground-hsl: #{get-hsl($fg-100)};
-  --iui-color-accent-hsl: var(--iui-color-informational-hsl);
-  --iui-color-informational-hsl: #{get-hsl($fg-informational-2)};
-  --iui-color-positive-hsl: #{get-hsl($fg-positive-2)};
-  --iui-color-warning-hsl: #{get-hsl($fg-warning-2)};
-  --iui-color-negative-hsl: #{get-hsl($fg-negative-2)};
+  // TODO: These need to be fixed
+  // --iui-color-accent-hsl: var(--iui-color-informational-hsl);
+  // --iui-color-informational-hsl: #{get-hsl($fg-informational-2)};
+  // --iui-color-positive-hsl: #{get-hsl($fg-positive-2)};
+  // --iui-color-warning-hsl: #{get-hsl($fg-warning-2)};
+  // --iui-color-negative-hsl: #{get-hsl($fg-negative-2)};
```

Comment:
The `-hsl` variables could probably be deprecated if we decide to move to `oklch`.

There's also [`color-mix`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix), which can completely replace these variables. For example:

```diff
  background-color:
-   hsl(var(--iui-color-accent-hsl) / var(--iui-opacity-5));
+   color-mix(in srgb, transparent,
+     var(--iui-color-background-accent) calc(var(--iui-opacity-5) * 100%));
```